operator/config/istio/ap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        paths:
+          - /metrics
```

Comment:
please include a comment for the common labels override

Reply:
I added the selector matchLabels, align with our deploy without kustomize decision.